debian/rules

```diff
@@ -48,15 +48,13 @@ override_dh_auto_install:
 	make -C apt-hook DESTDIR=$(CURDIR)/debian/ubuntu-advantage-tools install
 
 ifeq (${VERSION_ID},"14.04")
-	# Move ua-auto-attach.conf to ubuntu-advantage-pro
```

Comment:
I don't think we need any changes now in debian/rules now that no daemon /init present.